src/include/sof/audio/buffer.h

```diff
@@ -229,15 +229,14 @@ static inline void buffer_init(struct comp_buffer *buffer, uint32_t size,
 }
 
 static inline void buffer_copy_s16(struct comp_buffer *source,
-				   struct comp_buffer *sink, uint32_t bytes)
+				   struct comp_buffer *sink, uint32_t samples)
 {
-	uint32_t frames = bytes / sizeof(int16_t);
 	uint32_t buff_frag = 0;
 	int16_t *src;
 	int16_t *dst;
 	uint32_t i;
 
-	for (i = 0; i < frames; i++) {
+	for (i = 0; i < samples; i++) {
 		src = buffer_read_frag_s16(source, buff_frag);
```

Comment:
What is the difference between frames and samples?

Reply:
@dbaluta Wasn't the frame the collection of samples one per channel? As in, the frame being two samples in case of stereo?

Reply:
@dbaluta @paulstelian97 Exactly, the naming was wrong from the beginning.